plugins/services/src/js/containers/pod-instances/PodInstancesTable.js

```diff
@@ -363,16 +374,40 @@ class PodInstancesTable extends React.Component {
 
   renderColumnStatus(prop, row, rowOptions = {}) {
     let {status} = row;
+
     return this.renderWithClickHandler(rowOptions, (
-      <span>
-        <span className={status.dotClassName}></span>
-        <span className={`status-text ${status.textClassName}`}>
-          {status.displayName}
-        </span>
+      <span className={`status-text ${status.textClassName}`}>
+        {status.displayName}
       </span>
     ));
   }
 
+  renderColumnHealth(prop, row, rowOptions = {}) {
+    let {status} = row;
+    let {healthStatus} = status;
+    let tooltipContent = 'Healthy';
+
+    if (healthStatus === 'UNHEALTHY') {
+      tooltipContent = 'Unhealthy';
+    }
+
+    if (healthStatus === 'NA') {
+      tooltipContent = 'No health checks available';
+    }
+
+    return this.renderWithClickHandler(rowOptions, (
+      <div className="flex-box flex-box-align-vertical-center
+        table-cell-flex-box flex-align-items-center
+        flex-direction-top-to-bottom">
+        <div className="table-cell-icon">
+          <Tooltip anchor="center" content={tooltipContent}>
+            <span className={status.dotClassName}></span>
```

Comment:
We also need to add `flush` to this `span`, otherwise the tooltip won't be centered properly.

Also let's make it a self-closing element, e.g. `<span />`